http/src/client/mod.rs

```diff
@@ -1449,24 +1452,25 @@ impl Client {
         let mut builder = self.state.http.request(method.clone(), &url);
 
         if let Some(ref token) = self.state.token {
-            let value = HeaderValue::from_str(&token).map_err(|source| Error::CreatingHeader {
-                name: "Authroization".to_owned(),
-                source,
-            })?;
+            #[allow(clippy::borrow_interior_mutable_const)]
+            let name = AUTHORIZATION.to_string();
```

Comment:
Why is this built outside of the `.map_err()` below? I don't see another place its used and the old diff had it that way.  A block could be used inside to allow for the annoying clippy allow.

Reply:
Not sure. Fixed